packages/paste-core/components/alert/__tests__/index.spec.tsx

```diff
@@ -15,77 +13,57 @@ describe('Alert', () => {
   describe('Dismiss button', () => {
     it('Should add a dismiss button when onDismiss is passed as a function to call', () => {
       const eventHandlerMock: jest.Mock = jest.fn();
-      const wrapper: ReactWrapper = mount(
+      const {getByRole} = render(
```

Comment:
Not blocking, but you can call `screen.getByRole` instead of destructuing it from the render return.